book/website/reproducible-research/ci/ci-practices.md

```diff
@@ -1,5 +1,13 @@
 # Limitations and best practices
 
+```{figure} ../../figures/ContinuousIntegration.jpg
+---
+name: ContinuousIntegration
+alt: Illustration of continuous integration showing the lifecycle of continuous integration to plan, create and incorporate changes and updates to data. 
```

Comment:
My suggestion would be to expand the alt with: "Illustration of continuous integration showing the lifecycle of continuous integration to plan, create and incorporate changes and updates to data. The image shows these four steps in the lifecycle arranged in a circle and connected by arrows."